src/test/java/teammates/ui/webapi/CreateFeedbackSessionLogActionTest.java

```diff
@@ -21,12 +22,54 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO: test execute
+        JsonResult actionOutput;
```

Comment:
What happens if:

- Send request for feedback session that doesn't exist?
- Send request for email that doesn't exist?
- Send request for name that doesn't exist?
- Student is not supposed to have access to the feedback session?
- Student logs an access by a different student in the same session?

We don't have to solve all the cases, but we should consider them and what our expected result is.

Reply:
For each of these cases above, our current approach is to still carry on with logging down the information anyway: we don't perform rigorous validations (to avoid incurring costs for db reads and keep the endpoint light), so a success should be expected for all
Would it still be worth it to perform these different tests? It might not be that useful, but it could also be helpful to test what happens in the various situations for completeness?

Reply:
I see, let's have a tests for these as well so that in the future it is clear that these cases have been thought of and don't need special handling. Perhaps also add a comment (in the tests or in the action) on the reasoning.

Reply:
Added in the new tests too, and also a comment in the action for creating feedback session logs